AppCenter/AppCenter/Internals/Channel/MSChannelUnitDefault.m

```diff
@@ -229,6 +224,11 @@ - (void)flushQueue {
     return;
   }
 
+  // Ingestion is not ready.
+  if (!self.ingestion.isReadyToSend) {
+    return;
```

Comment:
This will keep all pending logs in the persistence and let it keep trying batch if there are logs more than 50. We should think about how we handle logs in this case.

Reply:
No, batches will have the same size, because `loadLogsWithGroupId:limit:withCompletion:` has limit parameter. The PR doesn't change this logic.

Reply:
I meant kind of an edge because it doesn't clean up logs. But I agree that this doesn't change any behaviors, we are going to rethink about this one later.